contrib/scalar/scalar.c

```diff
@@ -233,7 +502,7 @@ static int cmd_clone(int argc, const char **argv)
 
 	if (argc == 2) {
 		url = argv[0];
-		dir = xstrdup(argv[1]);
+		root = xstrdup(argv[1]);
```

Comment:
(Actually for the context above)

I think what you want is this:

```
	argc = parse_options(argc, argv, NULL, clone_options, clone_usage,
			     PARSE_OPT_KEEP_UNKNOWN);
```

Specifically, the Scalar functional tests present a different order:

`scalar clone <url> <dir> <other-options>`

Reply:
Oh... Good point. I shall change that, and adjust the regression test so that this is exercised.